outputs.tf

```diff
@@ -8,6 +8,11 @@ output "id_full" {
   description = "ID string not restricted in length"
 }
 
+output "id_camel_case" {
+  value       = local.enabled ? local.id_camel_case : ""
+  description = "Disambiguated ID string restricted to `id_length_limit` characters in total and with only alphanumeric characters"
```

Comment:
I think this should mention camel case also.

Reply:
Hey, I've added a usage example along with a link. Will this be sufficient ?